projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.ts

```diff
@@ -73,6 +73,12 @@ export class FormRendererComponent implements OnInit, OnChanges {
     if (this.parentComponent) {
       this.parentComponent.addChildComponent(this);
     }
+
+    if (this.node && this.node.question.renderingType === 'repeating' && this?.node?.question?.extras?.questionOptions?.min) {
+      for (let index = this.node.children.length; index < this?.node?.question?.extras?.questionOptions?.min; index++) {
```

Comment:
The `?` are redundant here and actually unhelpful...

```suggestion
      for (let index = this.node.children.length; index < this.node.question.extras.questionOptions.min; index++) {
```